include/seqan3/core/type_list/traits.hpp

```diff
@@ -234,6 +235,30 @@ auto split_after(type_list<pack1_t...>, type_list<head_t, pack2_t...>)
         return split_after<idx - 1>(type_list<pack1_t..., head_t>{}, type_list<pack2_t...>{});
 }
 
+//!\cond
+// Fall back that returns false if the unary function declaration is invalid for any given type.
+constexpr auto all_of(...)
+{
+    return false;
+}
+//!\endcond
```

Comment:
Not sure, what you mean but if I have something that does not SFINAE on an instantiation failure and is still ill-formed it will fail. It's only that you can use in addition SFINAE and do not provide a default case for every trait to test.